### PR TITLE
[luci/part] Revise append to hold virtual nodes

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise append method that prepares initial PGroup that
also holds virtual nodes for multiple outputs.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

File: compiler/luci/partition/src/PartitionPGroups.cpp [hunk 1]
```diff
@@ -236,22 +291,6 @@ std::unique_ptr<luci::PGroups> produce_pgroups(const luci::Module *source,
...
-  {
-    auto node = loco::must_cast<luci::CircleNode *>(nodes->at(idx));
-
-    // for virtual nodes like CircleUnpackOut should follow it's input (owner)
-    // or just set to default
-    FindGroupToFollow query(partition, pgroups.get());
-    const auto &group = node->accept(&query);
-    if (not group.empty())
-    {
-      append(node, pgroups.get(), group, idx);
```
Comment (seanshpark) [inline compiler/luci/partition/src/PartitionPGroups.cpp:251]: we don't need to append PGroup for virual nodes and append method itself will handle virtual nodes for multiple outputs

**Comment by seanshpark** [comment]
From #8004

**Comment by chunseoklee** [comment]
Please fix commit msg : thatprepares -> that prepares

**Comment by seanshpark** [comment]
I'll update commit message when I land the (to save time for another test run)

**Review by chunseoklee — APPROVED** [review]

**Comment by seanshpark** [comment]
log
```
commit 0c432964c37f04b98e020428d5f3896aee5f148f (HEAD -> master, ups/master, origin/master, origin/HEAD)
Author: SaeHie Park <saehie.park@gmail.com>
Date:   Thu Nov 18 11:14:12 2021 +0900

    [luci/part] Revise append to hold virtual nodes (#8022)

    This will revise append method that prepares initial PGroup that
    also holds virtual nodes for multiple outputs.

    ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>
```

**Pull request merged**